src/python/pants/report.py

```diff
@@ -0,0 +1,20 @@
+import uuid
+
+from humbug.consent import environment_variable_opt_out, HumbugConsent, no
+from humbug.report import Reporter
+
+BUGOUT_ACCESS_TOKEN: str = "3ae76900-9a68-4a87-a127-7c9f179d7272"
```

Comment:
We need to be aware that these can be used to spam our data, but I guess without end-user authentication that is inevitable.

Are there ways to make it a bit less obvious? Perhaps we can inject these at deploy artifact time, so at least someone has to reverse-engineer them.